apis/ec2/v1beta1/natgateway_types.go

```diff
@@ -0,0 +1,123 @@
+package v1beta1
```

Comment:
I have just realized that this is marked `v1beta1`. We prefer to keep the new resources on `v1alpha` level at least one release before bumping any resource to `v1beta1` so that we can make fixes easily without breaking the promise we make to the users.

Reply:
okay, that's fine. I should have read the docs, sorry for that. I'll move this over to `v1alpha4`. Does that sound good?

Reply:
No worries, all fine! I think we should start with `v1alpha1` and it doesn't need to go through many alpha versions. If everything looks good, we can consider bumping it directly to `v1beta1` in the next release.

Reply:
Sorry this took some time. I've been away and was taking some time off. I've move the resource into `v1alpha1`